components/script/dom/html/htmliframeelement.rs

```diff
@@ -84,6 +84,9 @@ pub(crate) struct HTMLIFrameElement {
     throttled: Cell<bool>,
     #[conditional_malloc_size_of]
     script_window_proxies: Rc<ScriptWindowProxies>,
+    /// Keeping track of whether the iframe will be navigated;
+    /// see the note in `iframe_load_event_steps`.
```

Comment:
can we be more specific with this doc commnent, since this flag only means "a real navigation (has already been triggered while we're still on the initial about:blank" ?